pkg/data/players_test.go

```diff
@@ -0,0 +1,27 @@
+package data
```

Comment:
Should add tests on individual functions AddPlayer, etc...
Will discuss on thursday meeting